skins/oasis/modules/LatestActivityController.class.php

```diff
@@ -58,6 +59,15 @@ public function executeIndex() {
 		$this->setVal( 'activityIcon', DesignSystemHelper::renderSvg( 'wds-icons-activity-small', 'wds-icon wds-icon-small wds-activity-icon' ) );
 		$this->setVal( 'renderCommunityEntryPoint', !empty( $wgEnableCommunityPageExt ) );
 
+		// TODO https://wikia-inc.atlassian.net/browse/CAKE-4746
+		if ( $wgEnableTriviaQuizzesExt && in_array($wgTitle->getText(), $wgTriviaQuizzesEnabledPages) ) {
```

Comment:
Similarly here as above, `$wgTitle->getText()` could be `$this->getContext()->getTitle()->getPrefixedText()`.

Reply:
Fixed in 93c4c1a7be7